src/fastoad/gui/tests/test_analysis_and_plots.py

```diff
@@ -126,6 +127,35 @@ def test_mass_breakdown_sun_plot():
     _ = mass_breakdown_sun_plot(filename)
 
 
+def test_mass_breakdown_sun_plot_specific_mission():
+    """
+    Test for testing the sun mass breakdown plotting for a specific mission.
+    """
+
+    filename = DATA_FOLDER_PATH / "problem_outputs_multi_mission.xml"
+
+    mission_1 = "evaluation_mission"
+
+    # Plot 1
+    # Specific mission plot
+    _ = mass_breakdown_sun_plot(filename, mission_name=mission_1)
+
+    mission_2 = "MTOW_mission"
+
+    # Plot 2
+    # Specific mission plot (sizing mission)
+    f = mass_breakdown_sun_plot(filename, mission_name=mission_2)
+    f.show()
```

Comment:
I guess you forgot to remove this line before the commit.
Note that I'm ok with commenting this line, so that a future dev will have just to uncomment it for seeing the result (same with plot 1).